opentype-shaping-arabic-general.md

```diff
@@ -252,51 +252,23 @@ JOIN_CAUSING:
     preceding character:
 	  - `isol` becomes `init`
 	  - `fina` becomes `medi`
-	  - `init` remains `init`
-	  - `medi` remains `medi`
   - If the current character's JOINING_TYPE is RIGHT, DUAL, or
     JOIN_CAUSING, tag the current character for `fina`, then update
     the tag for the preceding character:
 	  - `isol` becomes `init`
 	  - `fina` becomes `medi`
-	  - `init` remains `init`
-	  - `medi` remains `medi`
-  - If the current character's JOINING_TYPE is LEFT or NON_JOINING,
```

Comment:
Yep. LIke I've said in the past, I did find this bit confusing so thought it did need clarification. As to what wording for clarification, that's totally up to you. Take my diff more as a suggestion rather than a merge request :)